core/unix/os.c

```diff
@@ -295,6 +295,10 @@ static app_pc executable_end = NULL;
 static char executable_path[MAXIMUM_PATH];
 static char *executable_basename;
 
+/* Pointers to arguments. Refers to the main stack set up by the kernel. */
```

Comment:
Let's try to make new code more weak-memory-ordering (i.e., ARM) aware: let's put comments on globals as to why they do not need barriers on reads and writes.  Sthg like: these are only written once during process init and we can live with the non-guaranteed-delay until they are visible to other cores.

Reply:
Done.